src/apiCalls.js

```diff
@@ -0,0 +1,11 @@
+
+
+const fetchCalls = {
+  fetchData: function (endpoint) {
+    return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/${endpoint}`)
+      .then(response => response.json())
```

Comment:
This is a good place to check that the response is successful before trying to parse it. Check out `response.ok` for more on around that.